collator/src/lib.rs

```diff
@@ -333,6 +333,7 @@ pub async fn start_collator<P>(
 	para_id: ParaId,
 	key: Arc<CollatorPair>,
 	config: Configuration,
+	prefix: Option<String>,
```

Comment:
Prefix for what?

Reply:
True

Reply:
I renamed to `informant_prefix`. I think it's already quite self explanatory. Good?